lib/DDG/Util/ImageLoader.pm

```diff
@@ -0,0 +1,100 @@
+package DDG::Util::ImageLoader;
```

Comment:
I don't believe any IAs are actually using this? AFAIK it shouldn't be necessary anymore with the templates.

Let's deprecate this 😈